smoke-test/src/test/java/org/opennms/smoketest/minion/MinionHeartBeatTest.java

```diff
@@ -81,12 +87,24 @@ public void minionHeartBeatTestForLastUpdated() {
 		InetSocketAddress pgsql = m_testEnvironment.getServiceAddress(ContainerAlias.POSTGRES, 5432);
 		HibernateDaoFactory daoFactory = new HibernateDaoFactory(pgsql);
 		MinionDao minionDao = daoFactory.getDao(MinionDaoHibernate.class);
-
-		Criteria criteria = new CriteriaBuilder(OnmsMinion.class).ge("lastUpdated", startOfTest).toCriteria();
+		NodeDao nodeDao = daoFactory.getDao(NodeDaoHibernate.class);
 
 		// The heartbeat runs every minute so if we miss the first one, poll long enough
 		// to catch the next one
-		await().atMost(90, SECONDS).pollInterval(5, SECONDS).until(DaoUtils.countMatchingCallable(minionDao, criteria), greaterThan(0));
+		await().atMost(900, SECONDS)
+			   .pollInterval(5, SECONDS)
+			   .until(DaoUtils.countMatchingCallable(minionDao,
+													 new CriteriaBuilder(OnmsMinion.class).ge("lastUpdated", startOfTest).toCriteria()),
+					  greaterThan(0));
+
+		await().atMost(90, SECONDS)
+			   .pollInterval(5, SECONDS)
+			   .until(DaoUtils.countMatchingCallable(nodeDao,
+													 new CriteriaBuilder(OnmsNode.class).eq("foreignSource", "Minions@MINION")
```

Comment:
I'm guessing this will need to changed too, now that the default format has changed.